packages/strategy-openid/src/server/graphql/mutation/updateOpenIdAuthorities.ts

```diff
@@ -44,19 +47,22 @@ export const updateOpenIdAuthorities: GraphQLFieldConfig<
     }
   },
   async resolve(source, args, context): Promise<Promise<OpenIdAuthority>[]> {
-    const {
-      pool,
-      authorization: a,
-      realm,
-      strategies: { authorityMap }
-    } = context;
+    const { executor, authorization: a, realm } = context;
 
     if (!a) {
       throw new ForbiddenError(
         "You must be authenticated to update an authority."
       );
     }
 
+    const strategies = executor.strategies;
+    const pool = executor.connection;
+    if (!(pool instanceof Pool)) {
```

Comment:
No idea if this is a good idea, but it seems like this code fragment appears an awful lot. Could we have some kind of `executor.pool` getter?